services/galley/src/Galley/Data/Queries.hs

```diff
@@ -305,7 +305,7 @@ insertUserRemoteConv :: PrepQuery W (UserId, Domain, ConvId) ()
 insertUserRemoteConv = "insert into user_remote_conv (user, conv_remote_domain, conv_remote_id) values (?, ?, ?)"
 
 selectUserRemoteConvs :: PrepQuery R (Identity UserId) (Domain, ConvId)
-selectUserRemoteConvs = "select conv_remote_domain, conv_remote_id from user_remote_conv where user = ? order by conv_remote_domain"
```

Comment:
The order by clause would make it nonperforming, wouldn't it?

Reply:
The clause was redundant, our table is already ordered by `(conv_remote_domain, conv_remote_id)`.
https://github.com/wireapp/wire-server/blob/807c3e86eb32e4ed1d11abdd375dde8a4bb2908d/docs/reference/cassandra-schema.cql#L307-L313